swim/update_rollup_test.go

```diff
@@ -101,7 +103,7 @@ func (s *RollupTestSuite) TestFlushesAfterInterval() {
 	s.NotNil(s.r.FlushTimer(), "expected timer to be set")
 	s.Len(s.r.Buffer(), 2, "expected two updates to be in buffer")
 
-	time.Sleep(2 * time.Millisecond)
+	s.node.clock.(*clock.Mock).Add(2 * time.Millisecond)
```

Comment:
It's a good point. A related question is: how come the mock clock implementation doesn't just replace `Sleep` with the `Add`? That would make sense to me, because then the mock clock would work transparently, wouldn't it?